crates/rolldown/src/module_loader/module_task.rs

```diff
@@ -231,12 +239,13 @@ impl ModuleTask {
 
     let module_info = Arc::new(module.to_module_info());
     self.ctx.plugin_driver.set_module_info(&module.id, Arc::clone(&module_info));
-    self.ctx.plugin_driver.module_parsed(module_info).await?;
+    self.ctx.plugin_driver.module_parsed(Arc::clone(&module_info)).await?;
+    self.ctx.plugin_driver.mark_context_load_modules_loaded(&module.id).await?;
 
```

Comment:
Because we are running in parallel, is it possible that a task of `foo.js` is running in line 244 and stop here, then a `load(foo.js)` message send to `module_loader` thread so this callback would never be resolved.

Reply:
I guess we should check left fallbacks in the end of the module loader.

Reply:
Here `in the end of the module loader` couldn't happened, the `await this.load` always block it, we need to find a good pointer to check. Other way as https://github.com/rolldown/rolldown/pull/2687#issue-2648390432 said, we could refetch it again. I will have a try to resolve it at next.

Reply:
Oh. The js hook which calls `await this.load` would blocks the rust hook, so it never reachs the end of the module loader.